hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneFileSystem.java

```diff
@@ -457,6 +462,10 @@ public boolean delete(Path f, boolean recursive) throws IOException {
       return false;
     }
 
+    if (status == null) {
+      return false;
+    }
+
     String key = pathToKey(f);
     boolean result;
 
```

Comment:
Can we add javadoc to delete method here? parent abstract method delete does not provide any javadoc. SO, its good to provide here and explain the behavior?

Reply:
It complies to the `FileSystem#delete` behavior: https://github.com/apache/hadoop/blob/263c76b678275dfff867415c71ba9dc00a9235ef/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystem.java#L1634-L1643

But yes it would be a good idea to explain delete behavior for OFS.

Reply:
done. I've appended some extra javadoc for delete in 6d7a21b911b7b1ecff3a60443fa72782c5425de0